src/CLIMA.jl

```diff
@@ -1,6 +1,9 @@
 module CLIMA
 
-include("misc.jl")
+using CuArrays
+defaultarraytype() = CuArrays.functional() ? CuArray : Array
```

Comment:
Should this also check the environment flag like `init_default_array`?

Reply:
Ah, i meant to delete that.